cognite/client/config.py

```diff
@@ -70,13 +70,12 @@ def __init__(
         file_transfer_timeout: Optional[int] = None,
         debug: bool = False,
     ) -> None:
-        super().__init__()
         self.client_name = client_name
         self.project = project
         self.credentials = credentials
         self.api_subversion = api_subversion or __api_subversion__
         self.base_url = (base_url or "https://api.cognitedata.com").rstrip("/")
-        self.max_workers = max_workers if max_workers is not None else 10
+        self.max_workers = max_workers if max_workers is not None else 20
```

Comment:
Don't like this change. Most services don't currently allow this many concurrent requests. I'd prefer actually dropping this to 5 and rather spreading load in time.

Reply:
But keeping it at 10 is also fine.

Reply:
# 😞 
(Reverted to 10 in `dff5b78`)

Reply:
Note to self:
- [ ] Redo performance tests